File: src/test/fixtures/python/cyclomaticBranches.py
def classify(value):
    match value:
        case "a":
            return 1
        case "b":
            return 2
        case _:
            return 0


def classify_without_fallback(value):
    match value:
        case "a":
            return 1
        case "b":
            return 2

File: src/test/fixtures/fsharp/CyclomaticBranches.fs
// clean — not flagged by cyclomatic complexity
let classify value =
    match value with
    | "a" -> 1
    | "b" -> 2
    | _ -> 0

// clean — not flagged by cyclomatic complexity
let classifyWithoutFallback value =
    match value with
    | "a" -> 1
    | "b" -> 2
